halo2_proofs/src/plonk/circuit.rs

```diff
@@ -516,6 +591,7 @@ impl<F: Field> Expression<F> {
         &self,
         constant: &impl Fn(F) -> T,
         selector_column: &impl Fn(Selector) -> T,
+        virtual_column: &impl Fn(VirtualColumn) -> T,
```

Comment:
Since `Expression::evaluate` is public adding this argument is a breaking change.
Should I break out it into an extend variant of the function `Expression::evaluate_ext`, or is a major release planned so this breakage is fine?